db/migrate/20190518154841_add_permissions_to_users.rb

```diff
@@ -0,0 +1,19 @@
+class AddPermissionsToUsers < ActiveRecord::Migration[5.2]
+  def up
+    add_column :users, :permissions, :string, array: true, default: []
+    add_index :users, :permissions, using: 'gin'
```

Comment:
My general tendency is to try and be conservative with indexes (maybe that’s wrong; it certainly makes it easier to shoot yourself in the foot later) — and it doesn’t seem like we would use this one. Do we need it?

Reply:
Sounds good 👍 I removed them.